### ISSUE TITLE
Error: The container name "/database" is already in use by container 

### ISSUE THREAD

**Issue by Shelob9** [yukihiko-shinoda/docker-compose-phpunit-wordpress-plugin#1]
This is a very useful image. Thank you.

Sometimes when I start it, I get an error like this `ERROR: for database  Cannot create container for service database: Conflict. The container name "/database" is already in use by container "88314d275e3321b1e97b0de1a657da5f18f9722b5261f850d397d912b456ced3". You have to remove (or rename) that contai
ner to be able to reuse that name.`

I am then able to remove that container -- `docker rm 88314d275e3321b1e97b0de1a657da5f18f9722b5261f850d397d912b456ced3`  -- and start over and everything is fine. So, not a huge deal, but possible bug or maybe I'm wrong.

My docker-compose

```yml
version: '3.7'
services:
  database:
    container_name: database
    environment:
      MYSQL_ROOT_PASSWORD: examplepass
    image: mysql:5.7

  phpunit:
    command:
      - bash
    container_name: phpunit
    depends_on:
      - database
    environment:
      DATABASE_PASSWORD: examplepass
    image: futureys/phpunit-wordpress-plugin
    stdin_open: true
    tty: true
    volumes:
      - .:/plugin
```

**Comment by yukihiko-shinoda** [comment]
Thank you for finding and using this image!
And sorry! Compose file should be fixed.
If we already run other container named as "database", we will get its error.

I noticed after publish this repository that it was not good practice to specify `container_name` if we don't have any special reason.
Previously I specified it to use `docker exec` command since I didn't know `docker-compose exec`  command before.
cf.

- [docker-compose exec | Docker Documentation](https://docs.docker.com/compose/reference/exec/)
- [docker exec | Docker Documentation](https://docs.docker.com/engine/reference/commandline/exec/)

I'll fix Compose file:

```diff
version: '3.7'
services:
  database:
-    container_name: database
    environment:
      MYSQL_ROOT_PASSWORD: examplepass
    image: mysql:5.7

  phpunit:
    command:
      - bash
-    container_name: phpunit
    depends_on:
      - database
    environment:
      DATABASE_PASSWORD: examplepass
    image: futureys/phpunit-wordpress-plugin
    stdin_open: true
    tty: true
    volumes:
      - .:/plugin
```
